ow_lander/scripts/arm_move_joint_client.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import actionlib
+from actionlib_msgs.msg import GoalStatus
+from LanderInterface import JointStateSubscriber
+import ow_lander.msg
+import argparse
+from guarded_move_action_client import print_arguments
+from urdf_parser_py.urdf import URDF
+from distutils.util import strtobool
+
+
+def arm_move_joint_client():
+
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('relative', type=strtobool,
+                        help='Move joint relative to the current position', nargs='?', default='False', const=0)
+    parser.add_argument('joint', type=int,
+                        help='Joint index to be move 0:j_shou_yaw, 1:j_shou_pitch, 2:j_prox_pitch, 3:j_dist_pitch, 4:j_hand_yaw, 5:j_scoop_yaw', nargs='?', default=0, const=0)
```

Comment:
You can enforce a limited set of values using the `choices` kwarg.
```suggestion
    parser.add_argument('joint', type=int,
                        help='Joint index to be moved 0:j_shou_yaw, 1:j_shou_pitch, 2:j_prox_pitch, 3:j_dist_pitch, 4:j_hand_yaw, 5:j_scoop_yaw', nargs='?', 
                        default=0, const=0, choices=range(6))
```
This lets argparse handle invalid argument checks for you.